### PR TITLE
Path highlighting

### PR DESCRIPTION
**Pull request by cakey**
@Stewart-Taylor

### CODE REVIEW

File: src/client/app.coffee [hunk 1]
```diff
@@ -82,32 +88,15 @@ for hexX in [0..12]
         hexagons.add hexagon
         uuidToHex.set hexagon.uuid, [hexX, hexY]
         hexToUuid.set String([hexX, hexY]), hexagon.uuid
+        validHexes.add String([hexX, hexY])
 
 scene.add hexagons
 
-class GameState
```
Comment (Stewart-Taylor) [inline src/client/app.coffee:88]: Why did you removed GameState?

**Comment by Stewart-Taylor** [comment]
The pathing looks good :ghost:

File: src/client/app.coffee (hunk 1 above)
Comment (cakey) [inline src/client/app.coffee:88]: seemed like it was overlapping with gameView

we can factor it back out later (with view handling rendering/input), gameState being UI/client independent

Comment (Stewart-Taylor) [inline src/client/app.coffee:88]: okay

**Pull request merged**